src/org/usfirst/frc4904/robot/vision/AligningCamera.java

```diff
@@ -26,16 +27,16 @@ public AligningCamera() {
 		this(PIDSourceType.kDisplacement, AligningCamera.TABLE_NAME);
 	}
 
-	public float getDegrees() {
-		return (float) cameraTable.getNumber(AligningCamera.FIELD_DEGREES, Float.NaN);
+	public double getDegrees() {
+		return (float) cameraTable.getNumber(AligningCamera.FIELD_DEGREES, Double.NaN);
 	}
 
-	public float getDistance() {
-		return (float) cameraTable.getNumber(AligningCamera.FIELD_DISTANCE, Float.NaN);
+	public double getDistance() {
+		return cameraTable.getNumber(AligningCamera.FIELD_DISTANCE, Double.NaN);
 	}
 
 	public boolean isVisible() {
-		return cameraTable.getBoolean(AligningCamera.FIELD_VISIBLE, getDegrees() == Float.NaN || getDistance() == Float.NaN);
+		return cameraTable.getBoolean(AligningCamera.FIELD_VISIBLE, getDegrees() == Double.NaN || getDistance() == Double.NaN);
```

Comment:
Note that in Java, *nothing* is == to NaN, not even Double.Nan itself. `Double.NaN == Double.NaN` results in `false`.